client/src/Tickets/TicketTable.jsx

```diff
@@ -64,7 +45,7 @@ const TicketTable = ({ table, data, setData }) => {
               <td key={cell.id}>{flexRender(cell.column.columnDef.cell, cell.getContext())}</td>
             ))}
             <td>
-              <i className="bi bi-pencil pointer" onClick={onEdit(row)} />
+              <i className="bi bi-pencil pointer" onClick={() => navigate(`${row.original.id}`)} />
```

Comment:
Using ROUTES, we can open the modal just by navigating to the URL with the id in it.

This now also makes it possible to deep link to a specific ticket and pass that link around... before you could not...